package.json

```diff
@@ -11,15 +11,20 @@
 	},
 	"license": "ISC",
 	"dependencies": {
+		"-": "^0.0.1",
```

Comment:
Thank you for working on the pull request and adding the dependencies. I noticed that there are a couple of dependencies with invalid package names: "-" and "D". It seems like there might have been some errors or placeholders in the package.json file.